include/Database.php

```diff
@@ -85,8 +96,8 @@ public function connect()
         $db_user = $config['DB_USERNAME'];
         $db_password = $config['DB_PASSWORD'];
 
-        $pdo = new \PDO($dsn, $db_user, $db_password);
+        $pdo = new \PDO($dsn, $db_user, $db_password, array(\PDO::MYSQL_ATTR_FOUND_ROWS => true));
```

Comment:
I will insert a variable that will contain DSN driver specific array, instead of declaring directly here